packages/components/src/clipboard-button/index.js

```diff
@@ -15,6 +15,7 @@ import { useCopyOnClick } from '@wordpress/compose';
 import Button from '../button';
 
 export default function ClipboardButton( {
+	as: Component = Button,
```

Comment:
We can now remove this change, right? :)

Reply:
Yes we don't need it. but need help !, am i allowed to remove the commit containing this change?  daddou-ma/gutenberg@1be6587 or i should make another commit to remove it ?

Reply:
removed in daddou-ma/gutenberg@ebb7fee